formapp/templates/usermanual.html

```diff
@@ -221,5 +221,66 @@ <h2>Other Documents</h2>
+    <h3>VirtualBar</h3>
+    <p>The virtualbar will allow you to view a visual list of all poll answers, it will not display names.</p>
+</div>
+<u><h2>Games</h2></u>
+<div>
+    <p>The game section is filled with fun little mini-games you can play on your downtime. These were all made from scratch and some even include fun trivia about the class!</p>
+    <h3>Leaderboards</h3>
+    <p>The Leaderboard displays all the scores for each game and ranks them from best to worst.</p>
+</div>
+<u><h2>Teacher</h2></u>
+<div>
+    <h3>Teacher Controls</h3>
+    <p>The control panel is where teachers can view a list of all students, change user permissions, manage user profiles, view poll history and help/bathroom tickets, and full reboot the formbar. </p>
+    <h3>User Accounts</h3>
+    <p>This will allow teachers to change a students password per their request, delete their accounts, and reward them with digipogs. You just have to click the corresponding link and it'll guide you from there.</p>
+    <h4>Changing Passwords</h4>
+    <p>To change a user's password, after the input box comes up, you will enter the desired new password and hit "OK" have the student login once again.</p>
+    <h4>Adding and Removing Digipogs</h4>
+    <p>To add digipogs simply enter a static whole number and hit "OK" to remove them put a minus sign before the number to make a negative and then hit "OK".</p>
+    <h4>Lesson Creation</h4>
+    <p>Lessons are what you will use to teach each and every student in your class. You will utilize your controls, the lightbar, and more in each lesson. Creating a lesson is simple and easy.
+    Process and starting a new excel spreadsheet: Go into your excel in your computer, Start a blank excel sheet, Make two columns with word and definition respectively, Make the words you need in the word column, and Put the definition for those words in the definition column.
+    This is only for vocabulary, you will then create 6 more tabs.
+    These tabs will Quiz_1, Progress_1, Steps, Objectives, Resources, And Agenda.
+    quiz will be used for any quiz or test you might have in store for your students.
+    Progress will be used to measure how much you have done that class.
+    Steps will be used to get each and every step you will have your students do while they are working.
+    Objectives show all the things you want to get done that day.
+    Resources are all the external things you are going to use.
+    Agenda is what you want done and the time you want it done by.
+    Start by adding vocabulary and the various thnings needed in your lesson.
+    Uploading a lesson: Go to the Teacher tab in formbar, Go to the red lesson tab in the teacher tab, Click upload a lesson, Choose the excel spreadsheet file, and Confirm and upload.
```

Comment:
Did you test the formatting on this? This would be a huge wall of text. Fix the formatting and consider adding small pictures to help explain.

Go into detail about how to set up each category.